source/atlas-cli-changelog.txt

```diff
@@ -12,6 +12,50 @@
    :depth: 1
    :class: singlecol
 
+.. _atlas-cli-1.6.0:
+
+{+atlas-cli+} 1.6.0
+-------------------
+
+Released 30 March 2023
+
+- Adds the following new commands:
+
+  - :ref:`serverless-backups-snapshots-watch`
+  - :ref:`serverless-backups-restores-watch`
+  - :ref:`serverless-backups-restores-create`
+  - :ref:`serverless-backups-restores-describe`
+  - :ref:`atlas-serverless-backups-snapshots-list`
+  - :ref:`atlas-serverless-backups-restores-list`
+  - :ref:`atlas-serverless-update`
+  - :ref:`atlas-backups-exports-jobs-watch`
+
+- Adds the following to commands:
+
+  - ``file`` option for :ref:`atlas-backups-schedule-update` command.
+  - ``srvShardOptimizedConnectionString`` field to
+    ``privateEndpoints`` in
+    :ref:`atlas-clusters-connectionStrings-describe` command output. 
+  - Option to auto-detect the requester's current IP for the
+    :ref:`atlas-organizations-apiKeys-accessLists-create` command.
+
+- Adds the ability to create organizations.
```

Comment:
do we have a docs page that we can link to for this?

Reply:
@fmenezes could you please let us know which page to link to? Thanks!